apis/v1alpha2/grpcroute_types.go

```diff
@@ -391,10 +392,31 @@ type GRPCHeaderMatch struct {
 	Value string `json:"value"`
 }
 
+// GRPCHeaderMatchType specifies the semantics of how GRPC header values should
+// be compared. Valid GRPCHeaderMatchType values are:
+//
+// * "Exact" * "RegularExpression"
+//
+// Note that values may be added to this enum, implementations must ensure that
```

Comment:
I can see this language potentially being confusing for non-native english speakers, what do you think about:

```suggestion
// Note that new values may be added to this enum in future releases, implementations MUST ensure that
```

Reply:
Updated, thanks!